infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/utils/LogReplicationConfigManager.java

```diff
@@ -99,14 +89,26 @@ public LogReplicationConfigManager(CorfuRuntime runtime, String pluginConfigFile
         this.corfuStore = new CorfuStore(runtime);
 
         initStreamNameFetcherPlugin();
-        openTables();
+        openVersionTable();
+        // TODO: delete this method after upgrading
+        unregisterStreamsNameTable(runtime);
+    }
+
+    // Temporary helper method for remove Streams_Name_Table from registry table, for verticals
+    // already have that table registered.
+    private void unregisterStreamsNameTable(CorfuRuntime runtime) {
+        TableName streamsNameTable = TableName.newBuilder()
+                        .setTableName("LogReplicationStreams")
+                        .setNamespace(CORFU_SYSTEM_NAMESPACE)
+                        .build();
+        if (runtime.getTableRegistry().getRegistryTable().containsKey(streamsNameTable)) {
+            log.debug("Deleting record of Streams_Name_Table from registry table");
+            runtime.getTableRegistry().getRegistryTable().delete(streamsNameTable);
```

Comment:
We should have a test for this, which also verifies that the checkpoint just drops this table. It should emulate the upgrade scenario in an easy way.